app/javascript/src/components/Dashboard/Notes/NoteTable.jsx

```diff
@@ -26,8 +43,13 @@ export default function NoteTable({
                 }}
               />
             </th>
-            <th className="text-left">Title</th>
-            <th className="text-left">Description</th>
+            <th className="text-left text-gray-400">title</th>
+            <th className="text-left text-gray-400">description</th>
+            <th className="text-center text-gray-400">tags</th>
+            <th className="text-center text-gray-400">created date</th>
+            <th className="text-center text-gray-400">due date</th>
+            <th className="text-center text-gray-400">contact</th>
```

Comment:
I guess we can remove `text-gray-400` from here.

Reply:
@karthiknmenon `text-gray-400` is used to match gray headings as per design. Do you still want me to remove it anyway?

Reply:
@vipul-goel Is there hover effects (background color to grey) on hovering over a row on the table?